app/src/main/java/com/poohcom1/playalong/ui/scenes/SongController.kt

```diff
@@ -21,21 +19,31 @@ import com.yausername.youtubedl_android.mapper.VideoInfo
 fun SongController(info: VideoInfo, modifier: Modifier = Modifier) {
     var start: Float by remember { mutableStateOf(0f) }
     var end: Float by remember { mutableStateOf(info.duration.toFloat()) }
+    var currentPosition: Long by remember { mutableStateOf(0L)}
 
     var loopRange: LongRange by remember { mutableStateOf(0L..info.duration * 1000) }
 
     Column(
         Modifier.padding(16.dp),
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
-        VideoPlayer(info.url!!, loopRange, 120f)
+        VideoPlayer(info.url!!, loopRange, 120f) { position ->
+            currentPosition = position
+        }
```

Comment:
This is a type error, since you changed the parameter type to `MutableState<Long>`.